spacy/util.py

```diff
@@ -1671,11 +1672,10 @@ def to_ternary_int(val) -> int:
 # See licenses/3rd_party_licenses.txt
 def packages_distributions() -> Dict[str, List[str]]:
     """Return a mapping of top-level packages to their distributions. We're
-    inlining this helper from the importlib_metadata "backport" here, since
     it's not available in the builtin importlib.metadata.
```

Comment:
Looks like you removed this line by accident?

Reply:
Haha don't worry about it, accidents happen and that what reviews are for! I would not have had a single comment otherwise, so at least now I feel useful ;-)